app/views/smart_answers/result.html.erb

```diff
@@ -1,4 +1,13 @@
 <% outcome = @presenter.current_node %>
+
+<% question_number = @presenter.collapsed_questions.count %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: @presenter.back_link(question_number)
```

Comment:
I think you meant for this to be a "/" in this commit (as a placeholder) and then the actual implementation in the next commit.

Reply:
Sorry I had some fear in rewriting the history and breaking everything as the code changed again in a later commit from the one I moved it to... the conflicts were not actually too bad... have sorted it out now.